source/_twig-components/functions/faker.function.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+$function = new Twig_SimpleFunction('faker', function ($string, $options="") {
+  $faker = Faker\Factory::create();
+  if ($options != "") {
```

Comment:
Why specifically check for an empty string? `!empty()` seems like it would cover more user error cases.